File: hw4.py
def main():
	x = 5;
	print(x);

	def sub1():
		y = x + 10;
		print(y);

		def sub2():
			z = (y+x) * 2;
			print(z);

			def sub3():
				sum = x + y + z;
				print(' x = '+ str(x) + '\n y = ' + str(y) +
				            '\n z = '+ str(z) +  '\n sum = ' + str(sum));
			sub3()

		sub2()

	sub1()

main()

File: hw4.js
function main() {
	var x = 5;
	console.log(x);

	function sub1() {
		var y = x + 10;
		console.log(y);

		function sub2() {
			var z = (y + x) * 2;
			console.log(z);

			function sub3() {
				var sum = x + y + z;
				console.log(sum);
				console.log(' x = '+ x + '\n y = ' + y +
				            '\n z = '+ z +  '\n sum = ' + sum);
			}
			sub3();
		}
		sub2();
	}
	sub1();
}
main();